test/setup/main.tf

```
@@ -16,7 +16,7 @@
 
 module "project" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 10.0.1"
+  version = "~> 11.3.1"
```

```suggestion
  version = "~> 13.0.0"
```
bump to newer version

---

can be done in another PR since it is not directly related to the change